modules/core/src/commonMain/kotlin/fr/acinq/lightning/wire/LightningMessages.kt

```diff
@@ -575,6 +575,7 @@ data class TxInitRbf(
     constructor(channelId: ByteVector32, lockTime: Long, feerate: FeeratePerKw, fundingContribution: Satoshi) : this(channelId, lockTime, feerate, TlvStream(TxInitRbfTlv.SharedOutputContributionTlv(fundingContribution)))
 
     val fundingContribution = tlvs.get<TxInitRbfTlv.SharedOutputContributionTlv>()?.amount ?: 0.sat
+    val requireConfirmedInputs: Boolean = tlvs.get<TxInitRbfTlv.RequireConfirmedInputsTlv>()?.let { true } ?: false
```

Comment:
this is never used so all I can do is check that the code is correct (it is). Not sure it's worth adding a test though ?